src/containers/scheduleExams/scheduleExams.jsx

```diff
@@ -16,20 +18,23 @@ const SchededuleExamsWrapper = ({getData}) => {
     } else {
       setData(null);
     }
-  }, [group]);
+  }, [group, getData]);
 
-  return (
-    data ?
-      <div style={{overflow: 'hidden'}}>
-        <GridWrapper>
-          <Header>
-            Розклад сесії для групи <WordAccent>{group.name}</WordAccent><br/>
-            на <WordAccent>перший семестр{' '}</WordAccent>
-          </Header>
-          {data && data.map((d, idx) => <Exam key={idx} data={d}/>)}
-        </GridWrapper>
-      </div> : null
-  );
+  return data ? (
+    <div style={{ overflow: "hidden" }}>
+      <GridWrapper>
+        <Header>
+          Розклад сесії для групи <WordAccent>{group.name}</WordAccent>
+          <br />
+          на{" "}
+          <WordAccent>
+            {currentWeek === "firstWeek" ? "перший семестр" : "другий семестр"}
```

Comment:
here semester is not actually depends on currentWeek, it should be calculated based on date.